README.md

```diff
@@ -1,73 +1,32 @@
-
-# upload a piece of data that will acts as input to the program
-$ ls ~/Desktop/videos
-video1.mov
-video2.mov
-
-$ nerd dataset upload ~/Desktop/videos
-Uploading dataset with ID 'd-96fac377'
-314.38 MiB / 314.38 MiB [=============================] 100.00%
-
-# start a workload
-# we start 2 workers that use the jrottenberg/ffmpeg container to work on the input dataset
-$ nerd workload start jrottenberg/ffmpeg
-    --instances 2
-    --input-dataset d-96fac377
-Started workload with ID 'w-96fac375'
-
-# create two tasks for this workload
-# this will start the jrottenberg/ffmpeg container twice with the given arguments
-# input dataset d-96fac377 will be available in /input, data in /output will be uploaded when the task has successfully executed
-$ nerd task create w-96fac375 -- -i /input/video1.mov -acodec copy -vcodec copy /output/video1.avi
-$ nerd task create w-96fac375 -- -i /input/video2.mov -acodec copy -vcodec copy /output/video2.avi
-
-# get status of tasks
-$ nerd task list w-96fac375
-TASK ID     CMD                                                                 OUTPUT ID   STATUS    CREATED
-t-14962176  [-i /input/video1.mov -acodec copy -vcodec copy /output/video1.avi] video1.avi  SUCCESS   1 minute ago
-t-89491732  [-i /input/video2.mov -acodec copy -vcodec copy /output/video2.avi]             PENDING   1 minute ago
-
-# when all tasks are done we can download the output
-$ nerd workload download w-96fac375 ~/Desktop/videos_out
-$ tree ~/Desktop/videos_out
-~/Desktop/videos_out
-├── 7691e3df0c824efc1007082057b9c867_14962176
-│   └── video1.avi
-└── f45371445a7b95f7352bf841c30e4f58_89491732
-    └── video2.avi
+[Nerdalize](https://www.nerdalize.com/) is building a different cloud. Instead of constructing huge datacenters, we're distributing our servers over homes. Homeowners use the residual heat for hot showers and to warm their house, and we don't need to build new infrastructure.
 
-```
+In order to make our cloud resources accessible and easy to use, we've developed a CLI that fits your workflow. Whether you’re a researcher, engineer or developer, it allows you to easily run your computations, simulations and analyses on our cloud infrastructure.
 
-Please note that each command has a `--help` option that shows how to use the command.
-Each command accepts at least the following options:
-```
---config-file=  location of config file [$NERD_CONFIG_FILE]
---session-file= location of session file [$NERD_SESSION_FILE]
---debug=  show debug output [$NERD_DEBUG]
---output=[pretty|raw|json] Format of the output (default: pretty)
-```
+__Features__:
+  - Moving __datasets__ from you workstation to the cloud and back is included right into the workflow
+  - Nerd ensures efficient and quick datatransfers through a __deduplication__ algorithm
+  - Send in __thousands of jobs__, the Nerd makes your resources are used as efficient as possible
+  - Package your software using the industry-standard __Docker__ containers
```

Comment:
using the industry-standard -> using industry-standard (? makes more sense to talk about "containers" than to talk about "the containers", no?)